### PR TITLE
refactor: client store code style

### PR DESCRIPTION
**Pull request by TheAlexLichter**
related: #325

### CODE REVIEW

**Comment by manniL** [comment]
@galvez Haven't accomplished that much by now as I started digging into VueX HMR recently.

**Comment by galvez** [comment]
@manniL @clarkdo I fixed conflcits and added the store directory to the restart watcher, I think it's the most sensible thing to do to enhance DX here. The issue of the CLI test remains though, we need to find a way to properly check for the interactive log in `unit/cli.test.js` (remember @clarkdo? the issue with `child_process.spawn`'s `on('data')`)

**Comment by manniL** [comment]
I think that's not it. I think the request was to [HMR modules directly](https://vuex.vuejs.org/guide/hot-reload.html) (not the whole page when they change :thinking:). However, it's a great step in the correct direction

File: lib/app/store.js [hunk 1]
```diff
@@ -3,86 +3,93 @@ import Vuex from 'vuex'
...
+  storeData = {}
 
-// If store is not an exported method = modules store
-if (typeof storeData !== 'function') {
-  // Store modules
-  if (!storeData.modules) {
-    storeData.modules = {}
-  }
+  // Recursive find files in {srcDir}/{dir.store}
+  files = require.context('@/<%= dir.store %>', true, /^\.\/(?!<%= ignorePrefix %>)[^.]+\.(<%= extensions %>)$/)
```
[the remark this answers is not in the document]
Comment (galvez) [reply lib/app/store.js]: @manniL `store` needed to be available in L130

**Comment by galvez** [comment]
@pi0 @manniL @clarkdo k, without the unnecessary HMR changes, which I removed, it's back to the original set of refactorings, minus unneeded comments, plus minor lint edits. Feel free to get this in as-is.

**Review by pi0 — APPROVED** [review]

File: lib/app/store.js [hunk 2]
```diff
@@ -3,20 +3,15 @@ import Vuex from 'vuex'
 
 Vue.use(Vuex)
 
-// Recursive find files in {srcDir}/{dir.store}
 const files = require.context('@/<%= dir.store %>', true, /^\.\/(?!<%= ignorePrefix %>)[^.]+\.(<%= extensions %>)$/)
```
Comment (manniL) [inline lib/app/store.js]: BTW: Is there an "ES6" equivalent for `require.context`?

Comment (pi0) [reply lib/app/store.js]: Let's keep this enhancement in another PR :) I will work on it

**Pull request merged**